rest/views.py

```diff
@@ -314,31 +299,47 @@ class MetacellLinkViewSet(BaseReadOnlyModelViewSet):
 
 
 @extend_schema(
-    summary="List gene expression per single cell",
-    tags=["Single cell", "Gene"],
+    summary="List gene expression per gene in a dataset",
+    tags=["Single cell"],
+    request=serializers.SingleCellGeneExpressionSerializer,
     parameters=[
         OpenApiParameter(
-            "genes",
-            str,
-            description=filters.SingleCellGeneExpressionFilter()
-            .base_filters["genes"]
-            .label,
-            examples=[
-                OpenApiExample(
-                    "Example", value="Transcription factors,Pkinase,Tadh_P33902"
-                )
-            ],
-        )
+            "gene",
+            int,
+            "query",
+            True,
+            "gene id",
+            examples=[OpenApiExample("Example", value="276096")],
+        ),
+        OpenApiParameter(
+            "dataset",
+            int,
+            "query",
+            True,
+            "dataset id",
+            examples=[OpenApiExample("Example", value="13")],
+        ),
     ],
 )
-class SingleCellGeneExpressionViewSet(BaseReadOnlyModelViewSet):
-    """List gene expression data per single cell."""
+class SingleCellGeneExpressionViewSet(viewsets.GenericViewSet):
+    """List the Single Cell Expression per Gene in a Dataset"""
 
-    queryset = models.SingleCellGeneExpression.objects.prefetch_related(
-        "single_cell", "gene", "gene__domains", "metacell", "metacell__type"
-    )
+    http_method_names = ["get"]
     serializer_class = serializers.SingleCellGeneExpressionSerializer
-    filterset_class = filters.SingleCellGeneExpressionFilter
+    queryset = models.SingleCellGeneExpression.objects.none()
+    filterset_class = None
+    pagination_class = None
+
+    def list(self, request, *args, **kwargs):
+        gene = request.GET.get("gene")
+        dataset = request.GET.get("dataset")
```

Comment:
This should be obtained from `filter`. Not sure if there is an easy way to do this.